python/instrumentation/openinference-instrumentation-langchain/src/openinference/instrumentation/langchain/_tracer.py

```diff
@@ -376,6 +376,8 @@ def _input_messages(
         parsed_messages.append(dict(_parse_message_data(first_messages.to_json())))
     elif hasattr(first_messages, "get"):
         parsed_messages.append(dict(_parse_message_data(first_messages)))
+    elif isinstance(first_messages, tuple) and len(first_messages) == 2:
+        parsed_messages.append( { MESSAGE_ROLE: first_messages[0], MESSAGE_CONTENT: first_messages[1] } )
```

Comment:
I've been trying to reproduce in a smaller example but couldn't yet so far.
However I was able to make this problem go away by replacing
```
            input = {
                "start_time": time.time(),
                "messages": [
                    (
                        "system",
                        get_system_prompt(),
                    ),
...
```
with
```
            input = {
                "start_time": time.time(),
                "messages": [
                    SystemMessage(
                        content=get_system_prompt(),
                    ),
...
```
🤔